scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
Thanks for sticking with me on this falcon.

You are right. In the go function, cp.currentSlideIndex can be used. Then index doesn't need to be passed in. I've added a [new commit](https://github.com/h5p/h5p-course-presentation/pull/40/commits/60bc186a6e03ace15baeeb0ec2c22db99dca8c7f) that has this.

Let me know if there is anything else that needs to be addressed. Thanks!